lib/rspec-rails.rb

```diff
@@ -8,6 +8,12 @@ module RSpec
   module Rails
     # Railtie to hook into Rails.
     class Railtie < ::Rails::Railtie
+
+      # As of Rails 5.1.0 you can register directories to work with `rake notes`
+      if Gem::Requirement.new(">= 5.1.0").satisfied_by?(Gem::Version.new(::Rails.version))
```

Comment:
You can see our preferred form a few lines down in fact, so making this `::Rails::VERSION::STRING >= '5.1'` would be preferred :)